include/spdk/bdev.h

```diff
@@ -242,6 +243,16 @@ struct spdk_bdev_io {
 		} reset;
 	} u;
 
+	/** Error information from a device */
+	union {
+		struct {
```

Comment:
Please add another doc comment for the nvme branch of the union, something like "only valid when status is SPDK_BDEV_IO_STATUS_NVME_ERROR".